src/app/static/src/tests/root/actions.test.ts

```diff
@@ -288,7 +289,15 @@ describe("root actions", () => {
             }),
             projects: mockProjectsState({
                 currentProject: {name: "p1", id: 1, versions: []}
-            })
+            }),
+            hintrVersion: {
+                hintrVersion: {
+                    naomi: "v1",
+                    hintr: "v2",
+                    rrq: "v3",
+                    traduire: "v4"
+                }
+            }
```

Comment:
hintrVersion: mockHintrVersionState({
```suggestion
            hintrVersion: {
                hintrVersion: {
                    naomi: "v1",
                    hintr: "v2",
                    rrq: "v3",
                    traduire: "v4"
                }
            }
```
                hintrVersion: {
                    naomi: "v1",
                    hintr: "v2",
                    rrq: "v3",
                    traduire: "v4"
                }
            })